### ISSUE TITLE
Multi get not work (always empty dictionary)

### ISSUE THREAD

**Issue by sometime** [cnblogs/EnyimMemcachedCore#24]
This code always return empty dictionary (keys is IEnumerable<string>)
_memcachedClient.Get(keys);

**Comment by cnblogs-dudu** [comment]
Please provide sample codes that reproduce this issue.

**Comment by sometime** [comment]
```
    public class CacheLayer
    {
        readonly IMemcachedClient _memcachedClient;
        public CacheProvider(IMemcachedClient memcachedClient)
        {
            _memcachedClient = memcachedClient;
        }
        public Type Get<Type>(string key)
        {
            return _memcachedClient.Get<Type>(key);
        }

        public IDictionary<string, object> Get(IEnumerable<string> keys)
        {
            return _memcachedClient.Get(keys);
        }

        public void Set(string key, object value)
        {
            _memcachedClient.Store(StoreMode.Set, key, value);
        }
    }
```


in my code

```
var key1 = "test-key-1";
var key2 = "test-key-2"
var obj1 = new HashSet<int> { 1, 2 };
var obj2 = new HashSet<int> { 3, 4 };
cacheLayer.Set(key1, obj1);
cacheLayer.Set(key2, obj2);

var multiResult = cacheLayer.Get(new string [] { key1, key2 }); // multiResult is empty dictionary !!!

var result1 = cacheLayer.Get<HashSet<int>(key1); // success return obj1
var result2 = cacheLayer.Get<HashSet<int>(key2); // success return obj1
```
multiResult - every time is empty  dictionary, but result1 and result2 - success return obj1 and obj2 values

**Comment by sometime** [comment]
up

**Comment by cnblogs-dudu** [comment]
Fixed in EnyimMemcachedCore 2.1.0. Please use `cacheLayer.Get<HashSet<int>>` or `cacheLayer.GetAsync<HashSet<int>>`.
